source/mqtt_request_response.c

```diff
@@ -1050,23 +1052,715 @@ napi_value aws_napi_mqtt_request_response_client_submit_request(napi_env env, na
     return NULL;
 }
 
-napi_value aws_napi_mqtt_streaming_operation_new(napi_env env, napi_callback_info info) {
-    (void)info;
+///////////////////////////////////////////////////////////////////////////////////////////
+
+struct aws_request_response_streaming_operation_binding {
+    struct aws_allocator *allocator;
+
+    /*
+     * May only be accessed from within the libuv thread.
+     */
+    struct aws_mqtt_rr_client_operation *streaming_operation;
+
+    /*
+     * +1 from successful new -> termination callback
+     * +1 for every in-flight callback from client event loop thread to lib uv thread
+     */
+    struct aws_ref_count ref_count;
+
+    /*
+     * Single count ref to the JS streaming operation object.
+     */
+    napi_ref node_streaming_operation_ref;
+
+    /*
+     * Single count ref to the node external managed by the binding.
+     */
+    napi_ref node_streaming_operation_external_ref;
```

Comment:
I don't see how this field is initialized. Is it added for future or am I just missing something?

Reply:
Fixed =/